driver-core/src/main/java/com/datastax/driver/core/exceptions/BootstrappingException.java

```diff
@@ -44,6 +45,15 @@ private BootstrappingException(InetSocketAddress address, String message, Bootst
     }
 
     /**
+     * The host that was bootstrapping.
+     *
+     * @return The host that was bootstrapping.
+     */
+    public InetAddress getHost() {
```

Comment:
These could use the `@Override` annotation, and then wouldn't need javadoc (since they'd inherit it from the parent, applies to a number of exception classes here).  Although I see some of the javadoc is specific to the class (although some of that documentation could be implicit based on the class documentation).